### PR TITLE
Change python path for smoketests

### PR DESCRIPTION
**Pull request by adiaaida**

### CODE REVIEW

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

File: perf.groovy [hunk 1]
```diff
@@ -36,11 +36,13 @@ def static getOSGroup(def os) {
                         def architecture = arch
                         def jobName = isSmoketest ? "perf_perflab_${os}_${arch}_${opt_level}_${jit}_smoketest" : "perf_perflab_${os}_${arch}_${opt_level}_${jit}"
                         def testEnv = ""
+                        def python = "py"
 
                         def newJob = job(Utilities.getFullJobName(project, jobName, isPR)) {
                             // Set the label.
                             if (isSmoketest) {
                                 label('Windows.Amd64.ClientRS4.DevEx.15.8.Perf')
+                                python = "C:\\python3.7.0\\python.exe"
```
Comment (tannergooding) [inline perf.groovy]: Is there a plan to ensure that it is on the path the next time the image is updated?

Comment (adiaaida) [reply perf.groovy]: I don't actually know how to make that happen, to be honest. This also seems not to have worked as I expected it to, so now I'm actually a little confused about where python has been installed.

Comment (tannergooding) [reply perf.groovy]: Can we remote into one of the machines and determine where it was installed?

Comment (adiaaida) [reply perf.groovy]: I'm creating a vm in the test lab to just that, it's just taking a while to create. I was going to remote into one of the machines created for one of the jobs, but it disappeared really quickly.

**Comment by ViktorHofer** [comment]
@adiaaida I'm trying to fix the perf runs in corefx but am confused. Which python version is now installed on the machines? 3.5 or 3.7?

**Pull request closed without merging**